synapse/lib/stormtypes.py

```diff
@@ -9368,3 +9378,46 @@ async def tobuidhex(valu, noneok=False):
         raise s_exc.BadCast(mesg=mesg)
 
     return valu
+
+async def totype(valu, basetypes=False) -> str:
+    '''
+    Convert a value to its Storm type string.
+
+    Args:
+        valu: The object to check.
+        basetypes (bool): If True, return the base Python class name as a fallback.
```

Comment:
I don't want to expose this option :)

Reply:
I don't expose it for storm users. There is a use case for this internally though in #3097